webapp/channels/src/components/admin_console/list_table/list_table.tsx

```diff
@@ -338,16 +338,14 @@ export function ListTable<TableType extends TableMandatoryTypes>(
                         />
                     </div>
                 )}
-
-                {/* TODO will implement next */}
-                {/* <Pagination
+                <Pagination
```

Comment:
please uncomment the top bar pagination code as well, since its required in the design